.github/workflows/test.yml

```diff
@@ -213,6 +213,34 @@ jobs:
 
       - name: Integration Tests
         run: pnpm -- turbo run integration-tests --filter=cli
+
+  go_e2e:
+    name: Go E2E Tests
+    needs: determine_jobs
+    if: needs.determine_jobs.outputs.go == 'true'
+    timeout-minutes: 30
```

Comment:
It looks like this might not be enough time to build and run the tests on Windows 😢

Reply:
it's the same as it is now? https://github.com/vercel/turbo/blob/01e143d80720b532369383e12c60f87368811185/.github/workflows/pr-go-e2e-filtered.yml#L20